### PR TITLE
update delivery service fields

### PR DESCRIPTION
**Pull request by ashish-timilsina**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1475/
Test FAILed.

File: docs/source/admin/traffic_ops/using.rst [hunk 1]
```diff
@@ -291,135 +291,149 @@ Delivery Service
...
+| Routing Name                                 | The routing name to use for the delivery FQDN, i.e. ``<routing-name>.<deliveryservice>.<cdn-domain>``. It must be a valid hostname without periods. Typical values would be ccr, edge and cdn. [2]_                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (limited) [inline docs/source/admin/traffic_ops/using.rst]: Might be useful to mention the default, I think its `tr`?

File: docs/source/admin/traffic_ops/using.rst [hunk 2]
```diff
@@ -291,135 +291,149 @@ Delivery Service
...
+|                                              |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+|                                              | - None: Do not limit by geography.
```
Comment (limited) [inline docs/source/admin/traffic_ops/using.rst]: Its now called CZF + Country Code(s)

Comment (limited) [inline docs/source/admin/traffic_ops/using.rst]: Could link here to the RGB Howto page

**Pull request merged**